src/include/daos_srv/iv.h

```diff
@@ -74,6 +74,7 @@ struct ds_iv_class {
 struct ds_iv_key {
 	d_rank_t	rank;
 	int		class_id;
+	daos_epoch_t	eph;
```

Comment:
This is pool IV specific, please move it from common ds_iv_key into pool_iv_key.

Reply:
ah, sure.